models/ed/R/write.configs.ed.R

```diff
@@ -267,19 +267,52 @@ write.config.ED2 <- function(defaults, trait.values, settings, run.id){
   # Get prefix of filename, append to dirname. 
   # Assumes pattern 'DIR/PREFIX.lat<REMAINDER OF FILENAME>' 
   # Slightly overcomplicated to avoid error if path name happened to contain '.lat'
-  prefix <- sub("\\.lat.*", "", basename(settings$run$inputs$site))
-  ed2in.text <- gsub('@SITE_PSSCSS@', 
-    file.path(dirname(settings$run$inputs$site),paste0(prefix, '.')), ed2in.text)
   
-  # Warning if css/pss specified in settings 
-  if(!all(
-    identical(dirname(settings$run$inputs$site), dirname(settings$run$inputs$css)),
-    identical(dirname(settings$run$inputs$site), dirname(settings$run$inputs$pss)),
-    identical(prefix, sub("\\.lat.*", "", basename(settings$run$inputs$css))),
-    identical(prefix, sub("\\.lat.*", "", basename(settings$run$inputs$css))) ))
-    logger.warn("ED2 css/pss/site files have different path+prefix, but only site path+prefix will be used")
+  # when pss or css not exists, case 0
+  if (is.null(settings$run$inputs$pss)|is.null(settings$run$inputs$css)){
+    ed2in.text <- gsub('@INIT_MODEL@', 0, ed2in.text)
+    ed2in.text <- gsub('@SITE_PSSCSS@', "", ed2in.text)
+  }
+  else {
+    # pss and css are not in the same directory, case 0
+    if (!identical(dirname(settings$run$inputs$pss), dirname(settings$run$inputs$css)))
+    {
+      ed2in.text <- gsub('@INIT_MODEL@', 0, ed2in.text)
```

Comment:
Better to treat this the same as if they have different prefix and use logger severe, since it might confuse the user. They specified a pss and css but the run will be treated as from 0.